platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/sync/protocol/SyncProtocolTests.java

```diff
@@ -229,8 +240,10 @@ void noReasonToInitiate() {
                 time);
 
         assertEquals(2, permitProvider.getNumAvailable());
-        assertFalse(protocol.shouldInitiate());
-        assertEquals(2, permitProvider.getNumAvailable());
+        // With the critical quorum disabled, this should now initiate
```

Comment:
I think this test can just be deleted, as it's just a duplicate of the `shouldInitiate` test

Reply:
test deleted